Tasks/CmdLine/task.loc.json

```diff
@@ -16,7 +16,7 @@
   "version": {
     "Major": 1,
     "Minor": 0,
-    "Patch": 8
+    "Patch": 10
```

Comment:
Yes, this change was done by @bryanmacfarlane and corresponding changes in the loc.json were missing. I feel this should be merged.